sidebars.js

```diff
@@ -10,12 +10,11 @@ module.exports = {
     {
       "Get started": [
       "overview",
-      "get-started/preparation",
       {
         "type": "category",
-        "label": "Install Okteto",
+        "label": "Install Okteto SH",
```

Comment:
I don't think we should put 'SH', it can lead to confusion (what is Okteto SH?). I think "Okteto" is fine, it's what we use across the docs and other content when talking about the platform.  And I think the guide is clear enough that if you are using SaaS, you can skip to the next step.